tables/competitions_extra.sql

```diff
@@ -25,7 +25,7 @@ CREATE TEMPORARY TABLE compresults
 SELECT competitionId, 
 		COUNT(DISTINCT personId) competitors, 
 		SUM((CASE WHEN regionalSingleRecord = 'WR' THEN 1 ELSE 0 END)+(CASE WHEN regionalAverageRecord = 'WR' THEN 1 ELSE 0 END)) WRs,
-		SUM((CASE WHEN regionalSingleRecord IN (SELECT recordName FROM wca_dev.continents WHERE id NOT LIKE '_Multiple%') THEN 1 ELSE 0 END)+(CASE WHEN regionalAverageRecord IN (SELECT recordName FROM wca_dev.continents WHERE id NOT LIKE '_Multiple%') THEN 1 ELSE 0 END)) CRs,
+		SUM((CASE WHEN regionalSingleRecord NOT IN ('','NR','WR') THEN 1 ELSE 0 END)+(CASE WHEN regionalAverageRecord NOT IN ('','NR','WR') THEN 1 ELSE 0 END)) CRs,
```

Comment:
I don't like this fix. I much much much prefer my method. A simple lookup of the continents table takes no time at all.